tests/integration.bats

```diff
@@ -474,78 +474,78 @@ setup_file() {
 }
 
 @test "flox switch-generation 9999" {
-  run $FLOX_CLI switch-generation 9999 -e $TEST_ENVIRONMENT
+  run $FLOX_CLI --bash-passthru switch-generation 9999 -e $TEST_ENVIRONMENT
   assert_failure
   assert_output --partial "ERROR: could not find environment data for generation '9999'"
 }
 
 @test "flox environments takes no arguments" {
-  run $FLOX_CLI environments -e $TEST_ENVIRONMENT
+  run $FLOX_CLI --bash-passthru environments -e $TEST_ENVIRONMENT
   assert_failure
-  assert_output --partial '`-e` is not expected in this context'
+  # assert_output --partial '`-e` is not expected in this context' # this is a bpaf error, cant expect that with --bash-passthru`
```

Comment:
Should we just drop this then?

Reply:
I want to make the smallest changes possible to the tests here.
I think its better to leave a note here why this assert was removed so we do remember to test for that if we eventually port this command (likely with different semantics but 🤷🏼).